apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/store/ValidatorConfigProvider.scala

```diff
@@ -8,10 +8,11 @@ import cats.implicits.{catsSyntaxOptionId, toBifunctorOps}
 import com.digitalasset.canton.logging.{NamedLoggerFactory, NamedLogging}
 import com.digitalasset.canton.topology.PartyId
 import com.digitalasset.canton.tracing.TraceContext
+import io.circe.Codec
```

Comment:
The changes in this file are unnecessary. If possible, try to avoid them as it only leads to noise in the git log and merge conflicts for people that happen to work on the same file.